Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG CUDA_VER=11.4.0
+ARG IMG_TYPE=base
+ARG LINUX_VER=ubuntu18.04
+FROM nvidia/cuda:${CUDA_VER}-${IMG_TYPE}-${LINUX_VER}
+
+ARG PY_VER=3.9
+ENV PATH=/opt/conda/bin:$PATH
+
+COPY --from=condaforge/mambaforge:4.12.0-0 /opt/conda /opt/conda
+RUN \
+  # ensure conda environment is always activated
+  ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh; \
+  echo ". /opt/conda/etc/profile.d/conda.sh; conda activate base" >> /etc/skel/.bashrc; \
+  echo ". /opt/conda/etc/profile.d/conda.sh; conda activate base" >> ~/.bashrc; \
+  # install expected Python version
+  mamba install -y python="${PY_VER}"; \
+  find /opt/conda -follow -type f -name '*.a' -delete; \
+  find /opt/conda -follow -type f -name '*.pyc' -delete; \
+  conda clean -afy
```

Comment:
Are we sure we want `-f` here?

Reply:
miniforge does it. do you see any issues with it?

- https://github.com/conda-forge/miniforge-images/blob/e994b96c3886e1514666360b499ddc940ad87598/ubuntu/Dockerfile#L38